index.html

```diff
@@ -8,7 +8,27 @@
 
   <body>
 
-    <!-- NAV BAR (this is to test using github/pull requests)-->
+    <section class="hero_container">
+
+        <div class="left-col">
+            <p class="Header_text">Welcome to the <span style="color:#7642BA">Greatest</span> Online Community!</p>
+            <p class="Sub_text">We are a rapidly growing group that started off on Meetup.com. Join us every Monday on our coding
+                adventures. Sign up to stay up to date on meetings and activities. We promise you’ll enjoy your stay at
+                BWD (Beginning Web Development)!</p>
+                <button class="cta_btn">
+                    <a href="https://www.meetup.com/beginning-web-development/" target="_blank">
+                        <div class="content">
+                            <img src="./assets/logo.png" class="logo">
```

Comment:
The name of "logo.png" should be changed so I know what logo it is based off of the file name alone. There also needs to be an "alt" attribute added so that screen readers can make better use of the image.